test/bin/pyutils/build_bootc_images.py

```diff
@@ -525,6 +529,21 @@ def process_group(groupdir, build_type, dry_run=False):
         common.close_junit()
 
 
+def build_single_image(dir, file, dry_run):
```

Comment:
Can we avoid duplicating this logic if we use optional file-name-filtering in the process_group function?